src/iidm/Identifiable.cpp

```diff
@@ -49,7 +50,11 @@ const std::string& Identifiable::getName() const {
 }
 
 const std::string& Identifiable::getProperty(const std::string& key) const {
-    return m_properties.get(key);
+    try {
+        return m_properties.get(key);
+    } catch (const stdcxx::Exception& error) {
+        throw PowsyblException(error.what());
+    }
```

Comment:
I think it's OK to not wrap the exception to a PowsyblException even if you have to change the tests